src/s3_handler.py

```
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+import boto3
+import os
+from src import config
+from boto3.session import Config
+
+def upload_to_s3(local_path,  dni="sin_dni"):
```

Ruff (0.12.2)

Missing return type annotation for public function `upload_to_s3`

Add return type annotation: `None`

(ANN201)